src/plugins/uploader.js

```diff
@@ -20,8 +20,8 @@ const inlineContentDispositionTypes = {
 	"audio/mpeg": "audio.mp3",
 	"audio/ogg": "audio.ogg",
 	"audio/vnd.wave": "audio.wav",
-	"audio/x-flac": "audio.flac",
-	"audio/x-m4a": "audio.m4a",
```

Comment:
These are compared to the return values of `file-path`, so this should break file-type detection unless `file-path` has changed what mime types it returns for these file types?